website_supplier/README.rst

```diff
@@ -0,0 +1,77 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+    
+================
+Website Supplier
+================
+
+This module was written to extend the functionality of the website 
+to allow user to publish supplier on the website.
+
```

Comment:
Reading the code realized that this app adds an extra menu on the footer to the new page `/suppliers/*` would be nice if you use plural for the word `supplier`, me as @rafaelbn  was a little confused,  I think something like 
```
"""
This module was written to extend the functionality of the website 
to allow users to publish his suppliers on a new page.
"""
```